puppet/modules/jenkins_job_builder/files/theforeman.org/unmanaged_jobs.rb

```diff
@@ -48,8 +48,8 @@ def main
 
   unmanaged_jobs = find_unmanaged_jobs_from_payload(payload)
 
-  unless unmanaged_jobs.empty?
-    `jenkins-jobs --conf #{inifile} delete --jobs-only #{format_jobs_for_output(unmanaged_jobs)}`
+  if Time.now.hour.zero? && !unmanaged_jobs.empty?
+    %x[jenkins-jobs --conf #{inifile} delete --jobs-only #{format_jobs_for_output(unmanaged_jobs)}]
```

Comment:
What was the reason for changing the invocation to `%x`?

Reply:
That's just a [style thing](http://rubocop.readthedocs.io/en/latest/cops_style/#enforcedstyle-percent_x) that rubocop usually enforces.